### PR TITLE
#106 add support for paragraph indentations

### PR DESCRIPTION
**Pull request by erenard**
This is an implementation of the proposed solution in the issue #106.

### CODE REVIEW

File: src/helpers/xml-builder.js [hunk 1]
```diff
@@ -895,6 +908,20 @@ const buildParagraph = (vNode, attributes, docxDocumentInstance) => {
...
     }
+    if (vNode.properties.style['margin-left'] || vNode.properties.style['margin-right']) {
+      const leftMargin = fixupMargin(vNode.properties.style['margin-left']);
+      const rightMargin = fixupMargin(vNode.properties.style['margin-right']);
+      const indentation = {}
+      if (leftMargin) {
+        indentation.left: leftMargin;
+      }
+      if (rightMargin) {
+        indentation.right: rightMargin;
```
Comment (privateOmega) [inline src/helpers/xml-builder.js:919]:
```suggestion
        indentation.right = rightMargin;
```

File: src/helpers/xml-builder.js [hunk 2]
```diff
@@ -895,6 +908,20 @@ const buildParagraph = (vNode, attributes, docxDocumentInstance) => {
           : null
       );
     }
+    if (vNode.properties.style['margin-left'] || vNode.properties.style['margin-right']) {
+      const leftMargin = fixupMargin(vNode.properties.style['margin-left']);
+      const rightMargin = fixupMargin(vNode.properties.style['margin-right']);
+      const indentation = {}
+      if (leftMargin) {
+        indentation.left: leftMargin;
```
Comment (privateOmega) [inline src/helpers/xml-builder.js:916]:
```suggestion
        indentation.left = leftMargin;
```

**Review by privateOmega — CHANGES_REQUESTED** [review]

**Comment by privateOmega** [comment]
right now buildIndentation method only takes in value for left identation, modify it to accept both left and right values and also set the defaults as 0, instead of 720.

**Comment by privateOmega** [comment]
@erenard Thanks for your contribution. :)

File: src/helpers/xml-builder.js (hunk 1 above)
Comment (erenard) [reply src/helpers/xml-builder.js]: Done !

File: src/helpers/xml-builder.js (hunk 2 above)
Comment (erenard) [reply src/helpers/xml-builder.js]: Done !

**Comment by erenard** [comment]
@privateOmega I fixed up everything and added two cases to the example.

**Comment by privateOmega** [comment]
@erenard Appreciate the changes, but when I tried switch target branch to develop, it messed everything up and I will be opening a different PR with the cherry-picked changes from this on develop.

**Pull request closed without merging**